devices/src/virtio/net.rs

```diff
@@ -1054,9 +1054,11 @@ mod tests {
             let ret = self.rx_single_frame();
             if self.rx.deferred_irqs {
                 self.rx.deferred_irqs = false;
-                self.signal_used_queue();
+                let _ = self.signal_used_queue();
+                ret
```

Comment:
Why duplicate `ret` on both branches of the `if` instead of leaving it as it was after the conditional block?

Reply:
Yea, good point; done.